### PR TITLE
Fix some minor bugs related to CPK

### LINKED ISSUE #333: [FE] Redeem Positions Results
From #306

The result page currently looks like this:

Screen Shot 2020-09-24 at 10 36 02

Maybe we could improve the results with some additional info.

For example: Report Payouts has an array with the payout's probabilities which we could show, but in this case we show nothing.

We should inform the redeemed amount for Redeem Positions.

### LINKED ISSUE #799: No tokens are returned to wallet when merge all positions with 'Use CPK Address' option switched on
Related to #735, #782

1. Switch on 'Use CPK Address' option
2. split positions for a condition (simple split)
3. Open merge positions page and  merge all the  just created positions
4. Check the wallet

**AR**: No tokens are returned to wallet

**ER**: Tokens are returned back to the wallet

### LINKED ISSUE #800: No tokens are returned to wallet when redeem positions with 'Use CPK Address' option switched on
Related to #735, #782

1. Switch on 'Use CPK Address' option
2. split positions for a condition (simple split)
3. Resolve a condition
4. Redeem a position under the just resolved condition
5. Check the wallet

**AR**: No tokens are returned to wallet

**ER**: Tokens are returned back to the wallet

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #800 
Closes #799 
Closes #333

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr807--conditionaltokensexplorer.review.gnosisdev.com

File: src/hooks/usePositionsList.tsx [hunk 1]
```diff
@@ -196,7 +196,13 @@ export const usePositionsList = (
...
         })
 
-        setData(Remote.success(positionListDataEnhanced))
+        const positionListDataEnhancedOrdered = positionListDataEnhanced.sort(
+          (
+            positionA: PositionWithUserBalanceWithDecimals,
+            positionB: PositionWithUserBalanceWithDecimals
+          ) => (+positionA.indexSets[0] > +positionB.indexSets[0] ? 1 : -1)
+        )
+        setData(Remote.success(positionListDataEnhancedOrdered))
```
Comment (mariano-aguero) [inline src/hooks/usePositionsList.tsx:205]: We sort by indexSet

**Review by hpmaxi — APPROVED** [review]

**Review by elena-zh — COMMENTED** [review]
HI @mariano-aguero !

The changes, related to the #799, #800 and #333 look good to me there. 
However, I cannot approve the PR due to sorting in the positions list and positions sections is broken: I do not know in which order it works now, but exactly not in 'Created at' desc. 
Could you please fix it?

**Comment by mariano-aguero** [comment]
> HI @mariano-aguero !
> 
> The changes, related to the #799, #800 and #333 look good to me there.
> However, I cannot approve the PR due to sorting in the positions list and positions sections is broken: I do not know in which order it works now, but exactly not in 'Created at' desc.
> Could you please fix it?

@elena Can you verify again ? wait until the new preview appears as a new comment

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr807--conditionaltokensexplorer.review.gnosisdev.com

**Comment by elena-zh** [comment]
@mariano-aguero , now everything look good.

**Review by elena-zh — APPROVED** [review]

**Pull request merged**